### PR TITLE
Add missing fields and remove unused field

### PR DESCRIPTION
**Pull request by jforgue**

### CODE REVIEW

**Comment by jforgue** [comment]
screen shot 2018-06-11 at 3 41 25 pm

Looks more or less the same ass before.

File: app/views/bus_stops/_form.haml [hunk 1]
```diff
@@ -21,6 +18,18 @@
...
         %td= f.check_box :completed
+      %tr
+        %td= f.label :garage_responsible
+        %td= f.select :garage_responsible, %w[SATCo UMTS VATCo], include_blank: true
+      %tr
+        %td= f.label :state_road
+        %td= f.check_box :state_road
+      %tr
+        %td= f.label :date_stop_checked
+        %td= f.date_select :date_stop_checked, value: Date.today, disabled: true
```
Comment (dfaulken) [inline app/views/bus_stops/_form.haml]: @mboneil10 my interpretation is that these ought to be autofilled by the controller when (and only when) someone is marking the stop as completed.

Is that your understanding?

Comment (mboneil10) [reply app/views/bus_stops/_form.haml]: Yes, my fault I forgot to tell him to look at that issue.

Comment (mboneil10) [reply app/views/bus_stops/_form.haml]: @jforgue, in #20 we concluded to do what @dfaulken said above. You can remove `:date_stop_checked` and `:stop_checked_by`. Sorry I didn't realize the confusion earlier.

Comment (jforgue) [reply app/views/bus_stops/_form.haml]: Will do, no worries.

Comment (jforgue) [reply app/views/bus_stops/_form.haml]: Do we just want the fields removed from the form or the attributes removed from the model? I assume the later as we mention `paper_trail` in the issue.

Comment (Anbranin) [reply app/views/bus_stops/_form.haml]: As we talked about, both of these attributes can be covered with PaperTrail `versions` and `whodunnit`, so I'd remove the columns entirely at least for now.

Comment (dfaulken) [reply app/views/bus_stops/_form.haml]: We'd discussed that that could have performance implications, so I agree to use PaperTrail for now, but we should keep the current approach in our back pocket.

**Pull request merged**